scripts/jaseci.yaml

```diff
@@ -11,9 +11,61 @@ metadata:
   namespace: default
   name: jaseci-orc
 rules:
-  - apiGroups: ["v1", "apps"]
-    resources: ["pods", "deployments", "services", "deployments.apps"]
-    verbs: ["get", "watch", "list", "update", "patch", "create"]
+  - apiGroups: [
```

Comment:
@amadolid You can ignore the changes in this file for now...what happened was I was using this yaml (not knowing there was another one in `scripts/default`) and none of kubernetes operations were working so I added the following apiGroups and resources definition but i think the one in default is the one we should all use now?

Reply:
Yiping to rever the changes in this.